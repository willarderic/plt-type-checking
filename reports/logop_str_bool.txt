TOKEN: LPAREN  LEXEME: (					TOKEN: LPAREN    LEXEME: (
TOKEN: AND  LEXEME: and						TOKEN: AND       LEXEME: and
LOG_OP -> and							LOG_OP -> and
BIN_OP -> LOG_OP						BIN_OP -> LOG_OP
TOKEN: STRCONST  LEXEME: "one"					TOKEN: STRCONST  LEXEME: "one"
CONST -> STRCONST						CONST -> STRCONST
EXPR -> CONST							EXPR -> CONST
TOKEN: T  LEXEME: t						TOKEN: T         LEXEME: t
CONST -> t							CONST -> t
EXPR -> CONST							EXPR -> CONST
ARITHLOGIC_EXPR -> BIN_OP EXPR EXPR				ARITHLOGIC_EXPR -> BIN_OP EXPR EXPR
PARENTHESIZED_EXPR -> ARITHLOGIC_EXPR				PARENTHESIZED_EXPR -> ARITHLOGIC_EXPR
TOKEN: RPAREN  LEXEME: )					TOKEN: RPAREN    LEXEME: )
EXPR -> ( PARENTHESIZED_EXPR )					EXPR -> ( PARENTHESIZED_EXPR )
START -> EXPR							START -> EXPR

---- Completed parsing ----					---- Completed parsing ----

