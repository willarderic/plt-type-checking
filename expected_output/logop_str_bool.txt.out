TOKEN: LPAREN    LEXEME: (
TOKEN: AND       LEXEME: and
LOG_OP -> and
BIN_OP -> LOG_OP
TOKEN: STRCONST  LEXEME: "one"
CONST -> STRCONST
EXPR -> CONST
TOKEN: T         LEXEME: t
CONST -> t
EXPR -> CONST
ARITHLOGIC_EXPR -> BIN_OP EXPR EXPR
PARENTHESIZED_EXPR -> ARITHLOGIC_EXPR
TOKEN: RPAREN    LEXEME: )
EXPR -> ( PARENTHESIZED_EXPR )
START -> EXPR

---- Completed parsing ----

